lib/mri/webrick/utils.rb

```diff
@@ -63,22 +63,12 @@ def create_listeners(address, port)
       unless port
         raise ArgumentError, "must specify port"
       end
-      res = Socket::getaddrinfo(address, port,
-                                Socket::AF_UNSPEC,   # address family
-                                Socket::SOCK_STREAM, # socket type
-                                0,                   # protocol
-                                Socket::AI_PASSIVE)  # flag
-      last_error = nil
-      sockets = []
-      res.each{|ai|
-        begin
-          sock = TCPServer.new(ai[3], port)
-          port = sock.addr[1] if port == 0
-          Utils::set_close_on_exec(sock)
-          sockets << sock
-        rescue => ex
-          last_error  = ex
-        end
+      sockets = Socket.tcp_server_sockets(address, port)
```

Comment:
This should be wrapped in `# TruffleRuby: start` and `# TruffleRuby: end`, otherwise it's apt to be overwritten the next time we do a standard library upgrade.

Reply:
This is removing a patch we previously had, rather than introducing a new one.

Reply:
Ahh. Then someone else should have commented so I wouldn't be confused :-)